### ISSUE TITLE
Parsing sometimes fails

### ISSUE THREAD

**Issue by simeonschaub** [Nervengift/kvvliveapi-rs#2]
Some of the time `kvvliveapi::departures_by_stop_with_max` fails with the following error:
```
{ kind: Json(Error("premature end of input", line: 1, column: 153)), url: None }
```
although other times it works just fine.
I've narrowed it down to the function `parse_departure_time`, which doesn't seem to be always able to cope with the received JSON-data.
If I view the returned JSON in a browser or use the Python API, everything works fine.

**Comment by simeonschaub** [comment]
I think I figured it out. It seems like the API changed and now returns `"0"` instead of `"sofort"`. Should be solved by #1 .

**Comment by Nervengift** [comment]
fixed by #1
